tests/python/frontend/onnx/test_forward.py

```diff
@@ -681,6 +675,32 @@ def test_slice():
         x, x, starts=(0, 0), ends=(9223372036854775807, 9223372036854775807), axes=(0, 3)
     )
 
+    def test_slice_with_strides():
+        class SliceWithStrides(torch.nn.Module):
+            def forward(self, x):
+                return x[..., 0::2] + x[..., 1::2]
+
+        class SliceWithStrides2(torch.nn.Module):
+            def forward(self, x):
+                return x[0::2, 0::2] + x[1::2, 1::2]
+
```

Comment:
Why do this with Pytorch? Having this kind of thing in this file means that people have to install even more frameworks if they want to be running/testing ONNX TVM. Why not just construct the ONNX graph manually like most of the rest of the tests?

Reply:
yeah, I did this way because the bug report https://github.com/apache/incubator-tvm/issues/6441 came with PyTorch code and I was too lazy to manually write equivalent onnx :) I'll update this to remove pytorch dep.

Reply:
Fixed